qpc/messages.py

```diff
@@ -47,6 +47,9 @@
     ' "--auth" input values. Failed to add profile "%s"'
 PROFILE_ADDED = 'Profile "%s" was added'
 
+SERVER_CONFIG_HOST_HELP = 'host or ip address'
+SERVER_CONFIG_PORT_HELP = 'port number'
```

Comment:
`SERVER_CONFIG_PORT_HELP = 'port number; default=8000'`

Reply:
Added default port to help message as suggested